src/renderer/widgets/NumberInput.tsx

```diff
@@ -231,6 +231,16 @@ const NumberInput = ({
     handleSubmit(newValue);
   };
 
+  const onWheel = (e: React.WheelEvent) => {
```

Comment:
Same as below comment - can we add this to the `import { } from react` so this can be `WheelEvent` instead?